examples/event_handling/pong_sgskip.py

```diff
@@ -0,0 +1,50 @@
+"""
+========
+Pong GTK
+========
+
+A small game demo using GTK.
+
+"""
+from __future__ import print_function, division
+import time
+
+
+import matplotlib.pyplot as plt
+import pipong
```

Comment:
Sphinx has a...download directive that can be used to  link to pipong to make it explicit that this is  a local library. Did something like that for the unit examples.